runtime/doc/news.txt

```diff
@@ -147,6 +147,8 @@ The following new APIs and features were added.
     capabilities and static `server_capabilities`.
   • Added a new `anchor_bias` option to |lsp-handlers| to aid in positioning of
     floating windows.
+  • Added support for connecting to servers using named pipes (Windows) or
+    unix domain sockets (Unix) via |vim.lsp.rpc.domain_socket_connect()|
```

Comment:
```suggestion
    unix domain sockets (Unix) via |vim.lsp.rpc.domain_socket_connect()|.
```

Reply:
And you actually need to add a doc entry to that (you need to generate to docs via `./scripts/gen_vimdoc.py`).